pkg/storage/couchdb/couchdbstore.go

```diff
@@ -227,3 +231,79 @@ func (c *CouchDBStore) getDataFromAttachment(k string) ([]byte, error) {
 
 	return data, nil
 }
+
+// Iterator returns an iterator that can be used to iterate over all the stored key-value pairs.
+func (c *CouchDBStore) Iterator() storage.StoreIterator {
+	allDocs, err := c.db.AllDocs(context.Background(), kivik.Options{"include_docs": true})
```

Comment:
I'm not familiar with how kivik handles iterators, i.e. does it load all docs into memory or does it perform some sort of streaming from CouchDB? I'm worried about loading a million docs into memory.
Normally an iterator is created based on some query or range in order to limit the data that's returned.

Reply:
For what I need it for, I want all of the documents every time.

Reply:
How many do you expect there to be?

Reply:
For the demo, I think there may be only a couple. But I'm not sure if that's a safe assumption... @fqutishat

Reply:
@DRK3 we should use demo assumptions :).

Reply:
Per @troyronda's comment, I believe the demo will only have a couple, in which case iterating through all the documents shouldn't be an issue. However, I definitely see your point where it would be very slow when there's a lot of documents... in which case, perhaps it's possible to set up indexing in CouchDB specifically for this sort of thing, but this might be a bit tricky since EDV is store-agnostic. Perhaps this could be done after the demo? @troyronda @fqutishat @bstasyszyn

Reply:
@troyronda Did you mean to say should NOT be ... ?

Reply:
@bstasyszyn I messaged @troyronda and he said that I should NOT use demo assumptions, so I guess I'll need to assume that this should be production ready

Reply:
Yes - sorry, meant should NOT